index.html

```diff
@@ -119,42 +118,42 @@ <h1 class="">Hi, I'm Sai&nbsp;Tarun</h1>
 
 		<div id="About" class="container-fluid section">
 			<div class="row">
-				<div class="col-10 offset-1 col-md-8 offset-md-2 col-lg-3 offset-lg-2 brand-bl">
+				<div data-aos="fade-right" class="col-10 offset-1 col-md-8 offset-md-2 col-lg-3 offset-lg-2 brand-bl">
 					<h3 class="brand">About Me</h3>
 					<h2>Specialist by need, Generalist by heart</h2>
 					<p class="lead">I've worked with full-stack systems for <strong>web</strong> and <strong>mobile</strong>, using whatever suits the project best. "Best" is of course subjective, so I keep exploring new options such as Flutter and others as they come!</p>
 					<a href="#Work" class="btn btn-primary">
 						Check out my work
 					</a>
 				</div>
-				<div class="col-10 offset-1 col-md-8 offset-md-2 col-lg-3 offset-lg-0">
+				<div data-aos="fade-up" data-aos-delay="200" class="col-10 offset-1 col-md-8 offset-md-2 col-lg-3 offset-lg-0">
 					<img src="/assets/about-dp.jpg" alt="Kaniganti Sai Tarun" class="img-fluid">
 				</div>
 				<div class="col-10 offset-1 col-md-8 offset-md-2 col-lg-3 offset-lg-0 sans-serif">
-					<p>I've previously worked for <strong>Accenture</strong> before moving on to <strong>Small Planet</strong>, where I currently work remotely as an Android Developer, primarily using Flutter for some cool stuff ✨</p>
-					<p class="lead">I'm a Google India &amp; PyTorch Udacity Scholar, and currently under Bertelsmann Scholarship. I'm also a Student Mentor for <strong>Udacity</strong> and contribute actively to several open-source communities such as <strong>AnitaB.org</strong> and <strong>FOSSASIA</strong>.</p>
-					<p class="san-serif">As a student I've completed two internships, one with <strong>Shamgar Software Solutions</strong> and <strong>eBiz Solutions, LLC</strong> I've been a Campus Ambassador for <strong>GeeksForGeeks</strong> as well as for multiple college fests like <strong>Techfest, IIT-B</strong> and <strong>BITS-Pilani, Hyd Campus</strong>, amongst others.</p>
+					<p data-aos="fade-up" data-aos-delay="400">I've previously worked for <strong>Accenture</strong> before moving on to <strong>Small Planet</strong>, where I currently work remotely as an Android Developer, primarily using Flutter for some cool stuff ✨</p>
```

Comment:
@SAITARUN55 Do you want to add info about being a Udacity mentor or anything else. Might have to rethink this section layout.